cinema/serializers.py

```diff
@@ -78,3 +77,31 @@ class MovieSessionDetailSerializer(MovieSessionSerializer):
     class Meta:
         model = MovieSession
         fields = ("id", "show_time", "movie", "cinema_hall")
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MovieSession
+        fields = ["id", "show_time", "movie_title", "cinema_hall_name", "cinema_hall_capacity"]
```

Comment:
There is a duplicate definition of the `MovieSessionSerializer` class. It is already defined earlier in the file. Please remove the duplicate to avoid confusion and potential errors.